checks/hsts.go

```diff
@@ -0,0 +1,98 @@
+package checks
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+const (
+	StrictTransportSecurity = "Strict-Transport-Security"
```

Comment:
are these only set as consts for the prupose of reuse for testing? i think this pollutes the package space